Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -6981,14 +6981,21 @@ static RefPtr<CSSPathValue> consumeBasicShapePath(CSSParserTokenRange& args, Opt
             return nullptr;
     }
 
-    if (args.peek().type() != StringToken)
-        return nullptr;
+    auto pathValueFromStringView = [&](const StringView& view) -> RefPtr<CSSPathValue> {
+        SVGPathByteStream byteStream;
+        if (!buildSVGPathByteStreamFromString(view, byteStream, UnalteredParsing) || byteStream.isEmpty())
+            return nullptr;
+        return CSSPathValue::create(WTFMove(byteStream), rule);
+    };
 
-    SVGPathByteStream byteStream;
-    if (!buildSVGPathByteStreamFromString(args.consumeIncludingWhitespace().value(), byteStream, UnalteredParsing) || byteStream.isEmpty())
+    switch (args.peek().type()) {
+    case IdentToken:
+        return pathValueFromStringView(args.consumeAll().serialize());
```

Comment:
Do we want to `consumeAll()` here or just the current ident token (especially when the string token case only consume a single token) ?

Reply:
I believe we do because there will be multiple tokens to match the long list of path data commands.

Reply:
Isn't the case of multiple data commands handle by the `StringToken` case ? Can you have multiple command written as IdentToken (so without string delimiters) ?

Reply:
Yes, when the value comes internally from the SVG attribute parsing code.

Reply:
This code has been entirely removed since we now bypass style resolution altogether and manually create a `CSSPathValue` with the `SVGPathByteStream`.